observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerExtension.java

```diff
@@ -4,15 +4,24 @@
  */
 package org.wildfly.extension.micrometer;
 
+import org.jboss.as.version.Stability;
 import org.wildfly.subsystem.SubsystemConfiguration;
 import org.wildfly.subsystem.SubsystemExtension;
 import org.wildfly.subsystem.SubsystemPersistence;
 
 public class MicrometerExtension extends SubsystemExtension<MicrometerSubsystemSchema> {
+    static final Stability FEATURE_STABILITY = Stability.DEFAULT;
+
     public MicrometerExtension() {
         super(SubsystemConfiguration.of(MicrometerConfigurationConstants.NAME,
                         MicrometerSubsystemModel.CURRENT,
                         MicrometerSubsystemRegistrar::new),
                 SubsystemPersistence.of(MicrometerSubsystemSchema.CURRENT));
     }
+
+    @Override
+    public Stability getStability() {
+        return FEATURE_STABILITY;
+    }
```

Comment:
Is there a need for this since it returns the same value as the default?

Reply:
Actually, both this and the constant above are new here (and the constant above) are new and are both left over from the effort to go into wildfly-ee, so they can be removed. That said, since this never overrode `getStability()` (since it predates that utility), defaulted (har har) to DEFAULT.